src/Integrations/class-amp.php

```diff
@@ -154,18 +152,19 @@ public static function construct_amp_json(): string {
 	 * consists of the site's Site ID if that's defined, or an empty array
 	 * otherwise.
 	 *
+	 * @link https://docs.parse.ly/google-amp/
 	 * @since 3.2.0
 	 *
 	 * @return array<string, array<string, string>>
 	 */
 	public static function construct_amp_config(): array {
 		$options = self::$parsely->get_options();
 
-		if ( isset( $options['apikey'] ) && is_string( $options['apikey'] ) && '' !== $options['apikey'] ) {
+		if ( self::$parsely->site_id_is_set() ) {
 			return array(
 				'vars' => array(
 					// This field will be rendered in a JS context.
-					'apikey' => esc_js( $options['apikey'] ),
+					'apikey' => esc_js( $options['site_id'] ),
```

Comment:
We're still using `apikey` here in the left part of the assignment. Is this on purpose?

Reply:
Yes it is on purpose. Ref: https://github.com/Parsely/wp-parsely/pull/1244#discussion_r1046622380